connectors/src/resources/connector/confluence.ts

```diff
@@ -6,24 +7,29 @@ import {
   ConfluenceSpace,
 } from "@connectors/lib/models/confluence";
 import type {
+  ConnectorProviderModelResourceMapping,
   ConnectorProviderStrategy,
   WithCreationAttributes,
 } from "@connectors/resources/connector/strategy";
 import type { ConnectorResource } from "@connectors/resources/connector_resource";
 
-export class ConfluenceConnectorStrategy implements ConnectorProviderStrategy {
+export class ConfluenceConnectorStrategy
+  implements ConnectorProviderStrategy<"confluence">
+{
   async makeNew(
-    connector: ConnectorResource,
+    connectorId: ModelId,
     blob: WithCreationAttributes<ConfluenceConfiguration>,
     transaction: Transaction
-  ): Promise<void> {
+  ): Promise<ConnectorProviderModelResourceMapping["confluence"] | null> {
     await ConfluenceConfiguration.create(
       {
         ...blob,
-        connectorId: connector.id,
+        connectorId,
       },
       { transaction }
     );
+
+    return null;
```

Comment:
We need to return `null` until we have a proper resource for this configuration, is that correct?

Reply:
Good question. let me check

Reply:
yeah that method is typed `Promise<ConnectorProviderConfigurationResource | null>` and that type does not exist for confluence

Reply:
`ConnectorProviderConfigurationResource` is `BaseResource of a ConnectorProviderModelM`